dumbbot.py

```diff
@@ -11,17 +11,30 @@ async def on_ready():
 
 @bot.event
 async def on_message(ctx):
-	print('{} said:\"{}\" in #{}'.format(ctx.author.name, ctx.content, ctx.channel.name))
-
-	if ctx.content.startswith('http') and ctx.author.id != 380935311540355072 and ctx.guild.id == 379321436478636034:
-		channel = bot.get_channel(380028343611031565)
-		await channel.send(ctx.content)
-	elif (not ctx.content.startswith('http')) and ctx.channel.id == 380028343611031565 and not ctx.attachments:
-		await ctx.delete()
-	elif regexp.search(ctx.content.lower()):
-		await ctx.add_reaction('\N{EYES}')
-	
-	await bot.process_commands(ctx)
+	'''Don't react to a message sent by the bot'''
+	if ctx.author.id != 380935311540355072:
+
+		'''Print all messages into console'''
+		print('{} said:\"{}\" in #{}'.format(ctx.author.name, ctx.content, ctx.channel.name))
```

Comment:
You should check out f-strings. I personally find them a bit cleaner compared to .format().